addons/l10n_ec/data/l10n_latam.document.type.csv

```diff
@@ -0,0 +1,37 @@
+"id","sequence","code","name","report_name","internal_type","doc_code_prefix","country_id/id","active","l10n_ec_check_format"
```

Comment:
what is l10n_ec_check_format used for?

Reply:
Some documents must be checked by patterns 001-001-000000XXX, but other document doesn't have this requirement